lib/gollum/templates/editor.mustache

```diff
@@ -1,4 +1,4 @@
-<div id="gollum-editor" data-escaped-name="{{escaped_name}}" class="{{#is_create_page}}create{{/is_create_page}}{{#is_edit_page}}edit{{/is_edit_page}} {{#allow_uploads}}uploads-allowed{{/allow_uploads}}">
+<div id="gollum-editor" data-escaped-name="{{escaped_name}}" class="{{#is_create_page}}create{{/is_create_page}}{{#is_edit_page}}edit{{/is_edit_page}} {{#allow_uploads}}uploads-allowed{{/allow_uploads}} tex2jax_ignore">
```

Comment:
What does this `tex2jax_ignore` addition do?

Reply:
I had to add the MathJax JS to the editor, but that caused a small glitch as MathJax would try to render math in the gollum editor (which didn't succeed, but looked weird). :) `tex2jax_ignore` causes MathJax to ignore text in this element.

Reply:
Ok.